packages/atomic/src/components/common/generated-answer/generated-answer-common.tsx

```diff
@@ -91,6 +95,10 @@ export class GeneratedAnswerCommon {
     }
 
     if (isGenerating) {
+      if (this.props.collapsible) {
+        this.props.setIsCollapsed(true);
+      }
+
```

Comment:
Is there a reason why you wouldn't want the prop at all (either true or false).

Cuz this would be as straightforward as we could get. `this.props.setIsCollapsed(this.props.collapsible);`

Reply:
yes it should be true all the time for each new generated answer